content/docs/reference/downstream-mtls-settings.mdx

```diff
@@ -0,0 +1,331 @@
+<TabItem value="Kubernetes" label="Kubernetes">
+
+The Ingress Controller does not support these settings.
+
+</TabItem>
+</Tabs>
+
+### Examples {#crl-examples}
+
+```yaml
+downstream_mtls:
+  crl: LS0tLS1CRUdJTiBYNTA5IENSTC0tLS0tCk...
+```
+
+```yaml
+downstream_mtls:
+  crl_file: /path/to/CRLs.pem
+```
+
+```bash
+DOWNSTREAM_MTLS_CRL=LS0tLS1CRUdJTiBYNTA5IENSTC0tLS0tCk...
+```
+
+```bash
+DOWNSTREAM_MTLS_CRL_FILE=/path/to/CRLs.pem
+```
+
+## Enforcement Mode
+
+**Enforcement Mode** controls Pomerium's behavior when a client does not present a trusted client certificate. There are three options:
+
+1. `policy_with_default_deny` — Pomerium will serve an HTML error page when making a request to any user-defined route without a trusted client certificate. (This is the default behavior.)
+
+   ![Screenshot of invalid client certificate error page](./img/495-error-page.png)
+
+   Note that in this mode, client certificates are not required for internal Pomerium routes, such as the authenticate URL (if using the self-hosted authenticate service) or the /.pomerium/ details page.
+
+   Any requests without a trusted client certificate will be logged by the authorize service, with the deny reason `client-certificate-required` (if no certificate was presented) or `invalid-client-certificate` (if an untrusted or invalid certificate was presented).
+
+2. `policy` — Pomerium will not require client certificates for any routes unless explicitly required by a route policy.
+
+   This mode allows you to configure only certain routes to require a trusted client certificate. To configure a route to require client certificates, add a policy Deny rule with the `invalid_client_certificate` criterion. (See [Pomerium Policy Language](/docs/capabilities/ppl) for more information.)
+
+   As in the previous mode, Pomerium will serve an HTML error page for any routes where client certificates are required but no trusted certificate is presented, and likewise these requests will be logged by the authorize service.
+
+3. `reject_connection` — Pomerium will require a trusted client certificate during the initial TLS handshake. Any connection attempts that do not present a trusted client certificate will be rejected.
+
+   Users will see a browser default error page. For example:
+
+   ![Screenshot of Chrome's client certificate error page](./img/chrome-client-cert-error-page.png)
+
+   This matches the default behavior of many popular reverse proxies, such as Apache httpd and Caddy.
+
+   Note that in this mode, client certificates will be required even for internal Pomerium routes, such as the authenticate URL (if using the self-hosted authenticate service) and the /.pomerium/ details page. In particular, any upstream services that perform [JWT verification](/docs/capabilities/getting-users-identity) will not be able to access the default JWKS endpoint. You may need to host your own JWKS endpoint, or provide some other means of distributing the JWT verification key.
```

Comment:
confirm that /.pomerium/ is correct?

Reply:
I believe this is correct. I've added a little explanation about the /.pomerium/ page.